src/css/dark.css

```diff
@@ -458,7 +435,7 @@ a.button {
 }
 
 a.button:hover {
-  color: #1faaf2;
+  color: #1faaf2 !important;
```

Comment:
Try to avoid `!important` as it tends to spread, see if we can just add more specificity. (you can repeat class name to add more if you need `a.button.button:hover`